ocpmodels/trainers/forces_trainer.py

```diff
@@ -473,36 +383,16 @@ def train(self):
                             epoch=epoch - 1 + (i + 1) / len(self.train_loader),
                         )
                         if (
-                            val_metrics[
-                                self.config["task"].get(
-                                    "primary_metric",
-                                    self.evaluator.task_primary_metric["s2ef"],
-                                )
-                            ]["metric"]
+                            val_metrics[primary_metric]["metric"]
```

Comment:
One ~common bug here is that `primary_metric` assumes higher is better, which is not the case when using say `forces_mae`. Could we come up with a solution that works when `primary_metric` is either of `forces_mae` or `energy_force_within_threshold`?